db/migrate/20220510115548_add_cohort_npq_applications.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddCohortNPQApplications < ActiveRecord::Migration[6.1]
+  def change
+    add_column :npq_applications, :cohort, :integer, default: 2021, null: false
```

Comment:
Should this not be a join to `Cohort`?

Reply:
ok, i've changed this so it is now a `Cohort` object